google-cloud-storage/acceptance/storage/universe_domain_test.rb

```diff
@@ -0,0 +1,86 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "storage_helper"
+
+describe Google::Cloud::Storage do
+
+  # Universe Domain Test project Credentials
+  $test_universe_domain = ENV["TEST_UNIVERSE_DOMAIN"]
```

Comment:
Where are we setting these environment variables?

Reply:
as per our discussion on call we are not setting these env vars explicitly
the tests were passing without explicit declaration only one test was failing related to universe domain location

Reply:
shall I add the explicit declaration in integration.sh file?

Reply:
Yes. We won't be able to get their values otherwise.

Reply:
hi @bajajneha27 
still  have the following failure in one of my test case
I have set all the system vars in test.rb file only then also I guess test_location var is missing in env vars
please suggest how to proceed with this 

  1) Failure:
Google::Cloud::Storage#test_0001_creates a new bucket with universe_domain [acceptance/storage/universe_domain_test.rb:61]:
Expected: nil
  Actual: "US"

163 runs, 1302 assertions, 1 failures, 0 errors, 2 skips

Reply:
I'd check if I'm getting data from the secret manager. Try moving these variables to integration.sh and double all the key names / paths are same.

Reply:
I had tried putting theses in integration.sh
but after that none of the values were being fetched from env vars